src/main/resources/org/jpeek/xsd/index.xsd

```diff
@@ -22,6 +22,14 @@ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
 -->
+<!--
+@todo #134:30min Index.xsd: add `xsd:documentation` tags as per issue #134. We need to
```

Comment:
@llorllale Why are you adding these todos here? They have nothing to do with the ticket or the ``metric.xsl`` functionality, right?

Reply:
@amihaiemil fixed, they had nothing to do with the ticket